CHANGELOG.md

```diff
@@ -16,8 +16,7 @@ Now:
 
 ```js
 gheets.getWorksheet('ABC', 'xyz')
-  .then(result => {})
-  .catch(error => {});
+  .then(result => {}, error => {});
 ```
```

Comment:
IME the previous example was better. You *always* want a .catch() at the end of a promise chain.

If you use the two-callback form of `.then()`, and an error is thrown in the onSuccess callback, you'll never see it.